sale_exception_holidays_public/tests/test_sale_exception.py

```diff
@@ -20,11 +19,9 @@ def setUpClass(cls):
         cls.holiday_model.search([]).unlink()
 
         # Create holidays
-        today = fields.Date.today()
-        this_year = today.year
-        holiday_date = today + timedelta(days=10)
+        holiday_date = fields.Date.today()
```

Comment:
Well, I meant to keep this as it was as well :)

Reply:
This suggestion @EmilioPascual looks interesting! 😄